Resources/Prototypes/Corvax/Recipes/Reactions/drinks.yml

```
@@ -0,0 +1,159 @@
+
+- type: reaction
+  id: Boyarsky
+  reactants:
+    Vodka:
+      amount: 3
+    Grenadine:
+      amount: 2
+    Hotsauce:
+      amount: 1
+  products:
+    Boyarsky: 6
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)